clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/ClinvarMapper.scala

```diff
@@ -245,6 +246,122 @@ class ClinvarMapper(implicit msgCoder: Coder[Msg]) extends Serializable {
     }
   }
 
+  val vaTraitSetMappings = Map(
+    NonEmptyList.of("@ID") -> ClinvarConstants.IdKey,
+    NonEmptyList.of("@Type") -> Str("type")
+  )
+
+  val mapVaTraitSets: Mapper =
+    _.transform("Cleanup Variation Archive Trait Sets")(
+      _.map(mapFields(_, vaTraitSetMappings))
+    ).distinctBy(_.obj(IdKey).str)
+
+  val vaTraitMappings = Map(
+    NonEmptyList.of("@ID") -> Str("trait_id"),
+    NonEmptyList.of("@Type") -> Str("type")
+  )
+
+  val mapVaTraits: Mapper =
+    _.transform("Cleanup Variation Archive Traits") {
+      _.map { `trait` =>
+        val mapped = mapFields(`trait`, vaTraitMappings)
```

Comment:
```suggestion
        val mapped = mapFields(`trait`, traitMappings)
```

`mapFields` here will end up storing everything that isn't in `traitMappings` under the `content` field, which I don't think we want. We could either:
1. Add "dummy" mappings to the list above for the fields we need, to prevent them from being aggregated in content
2. Do the special processing before we call `mapFields`

I've been using strategy 1 elsewhere because `mapFields` makes a copy of the input message, so it's safe to mutate its output in whatever way we want.